code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -107,6 +111,24 @@
 			transport containing a nuclear fission explosive, although it is useless without the proper code and authorization disk. While the code was likely found in minutes, the only disk that \
 			can activate this explosive is on your station. Ensure that it is protected at all times, and remain alert for possible intruders."
 
+/datum/game_mode/nuclear/generate_credit_text()
+	var/list/round_credits = list()
+	var/len_before_addition
+
+	if((world.time-SSticker.round_start_time) < (FLUKEOPS_TIME_DELAY)) // If the nukies died super early, they're basically a massive disappointment
```

Comment:
What if the ops were speedrunning and _won_ in under 20 minutes (not everyone goes war ops, y'know)? The ops should only be labelled as "fluke ops" if they _lost_ in under 20 minutes, if they lost within 5 minutes of their ship first arriving on/at the stations z-level, or if the round ended in a humiliating syndicate defeat.